includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -658,11 +658,11 @@ public function update_fees( $order, $balance_transaction_id ) {
 				$fee_refund = ! empty( $balance_transaction->fee ) ? WC_Stripe_Helper::format_balance_fee( $balance_transaction, 'fee' ) : 0;
 				$net_refund = ! empty( $balance_transaction->net ) ? WC_Stripe_Helper::format_balance_fee( $balance_transaction, 'net' ) : 0;
 
-				/*current data fee*/
-				$fee_current = get_post_meta( $order_id, self::META_NAME_FEE, true );
-				$net_current = get_post_meta( $order_id, self::META_NAME_NET, true );
+				// Current data fee & net
```

Comment:
Need period at the end as commented before.

Reply:
in order to understand, you just ask me to add a point (.) at the end of the comment?

Reply:
Yes that is the coding standard.  See here ![](http://cld.wthms.co/w03oCT+)